app/components/navbar.js

```diff
@@ -0,0 +1,160 @@
+            <button
+              type="button"
+              className="inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 hover:text-white focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
+              aria-controls="mobile-menu"
+              aria-expanded="false"
+            >
+              <span className="sr-only">Open main menu</span>
+
+              <svg
+                className="block h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+                />
+              </svg>
+
+              <svg
+                className="hidden h-6 w-6"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke-width="1.5"
+                stroke="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  d="M6 18L18 6M6 6l12 12"
+                />
+              </svg>
+            </button>
+          </div>
+          <div className="flex flex-1 items-center justify-center sm:items-stretch sm:justify-start">
+            <div className="flex flex-shrink-0 items-center">
+              <img
+                className="block h-8 w-auto lg:hidden"
+                src="/tickets.png"
+                alt="Your Company"
+              />
+              <img
+                className="hidden h-8 w-auto lg:block"
+                src="/tickets.png"
+                alt="Your Company"
+              />
+              <h2 className="text-2xl">Ticket-App</h2>
+            </div>
+            <div className="hidden sm:ml-6 sm:block">
+              <div className="flex space-x-4">
+                <a
+                  href="#"
+                  className="bg-black text-white rounded-md px-2 py-1.5 mt-0.5 text-sm font-medium"
+                  aria-current="page"
+                >
+                  Home
+                </a>
+              </div>
+            </div>
+          </div>
+          <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0">
+            <div className="relative ml-3">
+              <div>
+                <button
+                  type="button"
+                  className="flex "
+                  id="user-menu-button"
+                  aria-expanded="false"
+                  aria-haspopup="true"
+                >
+                  <span className="sr-only">Open user menu</span>
+                  <img className="h-8 w-8" src="/cart.png" alt="" />
```

Comment:
El carrito full black con este tamaño opaca mucho. Cuando precises un icono busca en estos, son los oficiales de Tailwind: https://heroicons.com/
```suggestion
                  <img className="h-8 w-8" src="/cart.png" alt="" />
```